docs/en/Getting-Started.md

```diff
@@ -1,81 +1,365 @@
+following command:
+
+```bash
+$ ./bin/alluxio fs ls /
+```
+
+Unfortunately, we do not have any files in Alluxio. We can solve that by copying a file into
+Alluxio. The `copyFromLocal` shell command is used to copy a local file into Alluxio.
+
+```bash
+$ ./bin/alluxio fs copyFromLocal LICENSE /LICENSE
+Copied LICENSE to /LICENSE
+```
+
+After copying the **LICENSE** file, we should be able to see it in Alluxio. List the files in
+Alluxio with the command:
+
+```bash
+$ ./bin/alluxio fs ls /
+26.22KB   06-20-2016 11:30:04:415  In Memory      /LICENSE
+```
+
+The output shows the file that exists in Alluxio, as well as some other useful information, like the
+size of the file, the date it was created, and the in-memory status of the file.
+
+You can also view the contents of the file through the Alluxio shell. The `cat` command will print
+the contents of the file.
+
+```bash
+$ ./bin/alluxio fs cat /LICENSE
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/
+
+   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+...
+```
+
+With the default configuration, Alluxio uses the local file system as its UnderFileSystem (UFS). The
+default path for the UFS is `./underFSStorage`. We can see what is in the UFS with:
+
+```bash
+$ ls ./underFSStorage/
+```
+
+However, the directory doesn't exist! This is because by default, Alluxio will write data into
+Alluxio space only, and not to the UFS.
+
+However, we can tell Alluxio to persist the file from Alluxio space to the UFS. The shell command
+`persist` will do just that.
+
+```bash
+$ ./bin/alluxio fs persist /LICENSE
+persisted file /LICENSE with size 26847
+```
+
+Now, if we examine the local UFS again, the file should appear.
+
+```bash
+$ ls ./underFSStorage
+LICENSE
+```
+
+If we browse the Alluxio file system in the [master's web UI](http://localhost:19999/browse) we can
+see the LICENSE file as well as other useful information. Here, the **Persistence State** column
+shows the file as **PERSISTED**.
+
+## [AWS Bonus] Mounting in Alluxio
+
+Alluxio unifies access to storage systems with the unified namespace feature. Read the [Unified
+Namespace blog post](http://www.alluxio.com/2016/04/unified-namespace-allowing-applications-to-access-data-anywhere/)
+and the [unified namespace documentation](Unified-and-Transparent-Namespace.html) for more detailed
+explanations of the feature.
+
+This feature allows users to mount different storage systems into the Alluxio namespace and simply
+access the files across various storage systems through the Alluxio namespace seamlessly.
+
+First, we will create a directory in Alluxio to store our mount points.
+
+```bash
+$ ./bin/alluxio fs mkdir /mnt
+Successfully created directory /mnt
+```
+
+Next, we will mount an S3 bucket to Alluxio. We have a sample bucket to use for this guide.
```

Comment:
we should make it clearer whether we want them to use our sample or create their own.